### DISCUSSION TITLE
🌌 | Something new...

### DISCUSSION THREAD

**Discussion by EhsanFox** [SeTar-Bot/Setar-Bot#59] [Ideas]
Hey everyone 👋🏻<br>this is our first Github Discussion 🥇  <br>Make sure to share your idea's that would help setar to be  ✨ **Something new** ✨

Any ideas such as:
- new commands 🤖 
- new UI Look 👀 
- new API/Interaction for Developers 👨🏻‍💻 
- implementing new features 🧪 

or anything is welcomed in here,
Don't Be Shy, Share some more 😄

Also don't forget to [Join our Discord Community](https://discord.gg/7jgfP6j4Tc)

**Comment by im-parsa** [comment]
Hey @EhsanFox 👋🏻 

🌟 My ideas:

- 🔢 A customized playlist is something that is less supported by the discord bots and can be a good idea to improve the bot.

- 👍🏻 There is also a section for voting on top songs in the bot to increase community activity.

- 🎵 It may be possible to maximize the quality of music and make settings for it that can be customized.

- 📟 Also, the idea of a customized dashboard is also extremely useful and important for the bot.

**Reply by EhsanFox** [reply]
These all amazing ideas will be reviewed after we have the done basic setup of the work, which you can have a very good impact on by finishing the assigned jobs in [Projects Table](https://github.com/orgs/SeTar-Bot/projects/1) whenever you have the job to do so.

We would for sure consider these ideas to merge them into the main version of Setar 👍
